install/operator/build/Dockerfile

```diff
@@ -1,7 +1,16 @@
-FROM alpine:3.8
+FROM registry.access.redhat.com/ubi7/ubi-minimal:latest
```

Comment:
Is it intended to use `registry.access.redhat.com` for the upstream version of the operator?

Reply:
Yep. And encouraged.